src/cogent3/evolve/likelihood_function.py

```diff
@@ -896,8 +896,22 @@ def to_rich_dict(self):
                 edge_attr[edge]["length"] = None
 
         model = self._model.to_rich_dict(for_pickle=False)
-        alignment = self.get_param_value("alignment").to_rich_dict()
-        mprobs = self.get_motif_probs().to_dict()
+
+        aln_defn = self.defn_for["alignment"]
+        if len(aln_defn.index) == 1:
+            alignment = self.get_param_value("alignment").to_rich_dict()
+            mprobs = self.get_motif_probs().to_dict()
+        else:
+            alignment = {a["locus"]: a["value"] for a in aln_defn.get_param_rules()}
+            mprobs = self.get_motif_probs()
+            if len(alignment) == 1:
```

Comment:
de-indent this condition – it should not be nested since it should be dealt with by the condition at line 901